JobParseAndNormalize.md

```diff
@@ -0,0 +1,353 @@
+ - [Job Geography](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/Geography.md) <sup>(optional)</sup>
+ - [Job Level](https://github.com/cbdr/DataScienceAPIDocumentation/blob/master/JobLevel.md) <sup>(optional)</sup>
+ - Education Level
+ - Employment Level
+ - Contract Type
+ 
+
+Enrichments marked as <sup>__(optional)__</sup> must be requested using the `desired_enrichments` parameter, as they require additional calls to other services. For example, a client who only needs Job Title and Skills classifications could structure a Job Parse and Normalize request that would only retrieve these classifications. This parameter is documented in further detail below.
+
+The service is located at https://api.careerbuilder.com/core/parsing/normalizedjob. As usual, you will need OAuth core credentials to use this service. (*If you do not have these, please go [here](http://apitester.cbplatform.link/credentials) or email PlatformSoftware@careerbuilder.com to request core credentials.*)
+
+##Request Structure
+
+This service supports the HTTP GET and POST methods. Because Base64-encoded documents can be quite large, POST is encouraged for production use.
+
+The following parameters may be supplied in the query string (for HTTP GET) or form body (for HTTP POST):
+
+* **document** (Required) -- A .doc, .docx, .pdf, .rtf, .txt, .odt, .wps, and .pages documents given in a BASE64 encoded string.  Please note that the .pages format is not accepted by Textkernel; you will need to specify another parser.
+
+* **desired_enrichments** (Required) -- A comma-separated list, without spaces, of the desired normalization calls to perform on the results of the job parsing operation. 
+
+    This list of possible values for **desired_enrichments** and the enrichments the correspond to are as follows:
+    
+    |desired_enrichments | [Enrichment(s)](#Enrichments) |
+    | --- | --- |
+    | `company_norm` | Normalized Company Name |
+    | `geocoding` | Company Geography, Job Geography |
+    | `job_level` | Job Level |
+    | `job_title_carotene` | Carotene Job Title Classifications |
+    | `job_title_onet` | ONET Job Title Classifications |
+    | `skills` | Skills 4.0 |
+
+    For example, a request with a desired_enrichments value equal to `job_level,skills,job_title_onet,company_norm` would receive job level classifications, skills extractions, ONet job title classifications, and company normalizations. The API does not currently allow callers to request only certain versions of a classification service.
+    
+    The value "none" may be supplied to return none of the optional enrichments. At present, a request that does not include this parameter will receive all classifications; this is temporary behavior for the sake of backwards compatibility. Once all customers have started using the desired_enrichments parameter, its usage will become required, and requests excluding it will result in a 400 Bad Request response code.
```

Comment:
Everything in this paragraph starting from "At present" is no longer true -- we made desired_enrichments mandatory last quarter. Could you remove this from RPAN too please?